search-service/src/test/kotlin/com/egm/stellio/search/service/QueryServiceTests.kt

```diff
@@ -228,6 +229,8 @@ class QueryServiceTests {
             every { temporalEntityService.buildTemporalEntities(any(), any(), any(), any()) } returns emptyList()
 
             queryService.queryTemporalEntities(
+                // 2,
```

Comment:
never leave commented code (but anyway I think these two parameters should be added to `queryTemporalEntities`